sa/proto/sa.proto

```diff
@@ -437,3 +447,46 @@ message UpdateRegistrationKeyRequest {
   int64 registrationID = 1;
   bytes jwk = 2;
 }
+
+message RateLimitOverride {
+  int64 limitEnum      = 1;
+  string bucketKey = 2;
+  string comment    = 3;
+  google.protobuf.Duration period = 4;
+  int64 count      = 5;
+  int64 burst      = 6;
+}
+
+message AddRateLimitOverrideRequest {
+  RateLimitOverride override = 1;
+}
+
+message AddRateLimitOverrideResponse {
+  bool inserted = 1;
+  bool enabled = 2;
+}
+
+message SetRateLimitOverrideRequest {
```

Comment:
Per https://protobuf.dev/best-practices/api/#unique-protos, we should have separate request message types for the Enable and Disable methods.